index.html

```diff
@@ -0,0 +1,119 @@
+    </div>
+  </div>
+
+  <section class="container custom-sec">
+    <div class="hero">
+      <h2>Handcrafted, home-made masterpieces</h2>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <input type="email" name="email" id="email" placeholder="enter your email here">
+        <input type="submit" id="submit" class="btn btn-warning">Get Started</input>
+      </form>
+    </div>
+
+    <div class="feature" id="features">
+      <div>
+        <h1>Premium Materials
+        </h1>
+        <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your
+          purchase.
+        </p>
+      </div>
+      <div>
+        <h1>Fast Shipping
+        </h1>
+        <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if
+          you are
+          not satisfied.
+        </p>
+      </div>
+
+      <div>
+        <h1>Quality Assurance
+        </h1>
+        <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the
+          pitch of your
+          instrument.
+        </p>
+      </div>
+      <div class="how-it-works" id="how">
+        <iframe id="video" width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc"
+          title="Roman Carnival Overture Op. 9 for Five Trombones" frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+          allowfullscreen></iframe>
+      </div>
+    </div>
+  </section>
+
+  <section class="cards" id="cards">
+    <div class="card">
+      <h5 class="card-header">Featured</h5>
+      <div class="card-body">
+        <h5 class="card-title">Special title treatment</h5>
+        <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
+        <a href="#" class="btn btn-warning">Go somewhere</a>
+      </div>
+    </div>
+
+    <div class="card">
+      <h5 class="card-header">Featured</h5>
+      <div class="card-body">
+        <h5 class="card-title">Special title treatment</h5>
+        <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
+        <a href="#" class="btn btn-warning">Go somewhere</a>
+      </div>
+    </div>
+
+    <div class="card">
+      <h5 class="card-header">Featured</h5>
+      <div class="card-body">
+        <h5 class="card-title">Special title treatment</h5>
+        <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
+        <a href="#" class="btn btn-warning">Go somewhere</a>
+      </div>
+    </div>
+  </section>
+
+
+
+  <div>
+    <ul class="footers">
+      <li><a href="#features">Features</a></li>
+      <li><a href="#how">Terms</a></li>
+      <li><a href="#">How it works</a></li>
+    </ul>
+    <span>Copyright 2023, Himanshu and MoFuhidy</span>
+  </div>
```

Comment:
please change the div tag to a footer tag . for semantic reasons

Reply:
- [ ] As I mentioned above :point_up_2: please give more explanation about `semantic` tags then the student understand easily :slightly_smiling_face:

Reply:
A semantic element clearly describes its meaning to both the browser and the developer.

Reply:
Well, the last reviewer asked you to explain semantics a bit but now in the latest comment from your side, I can see the explanation of semantics without describing what the user needs to change.

### Why this happened and what to do now?

Well, actually what the last Code Reviewer meant was to recreate the comment or edit it by describing more clearly about semantics not only describe something related to semantics. So, consider kind of taking both of your comments making a good sentence from both of them and making another comment asking for a change with a descriptive reason, that's it. 💯

Reply:
please change the div tag to a footer tag . So that it can clearly define its meaning to both the browser and the developer.